### ISSUE TITLE
Ignored classes and nestedTensor

### ISSUE THREAD

**Issue by Kishaan** [conradry/max-deeplab#8]
Hi, 

Thank you for the implementation. I have two questions regarding the code. 
1. What is the significance of converting list of Tensors to nestedTensor in dataloader? 
2. Are class labels for the background 0 and 201 ignored in the loss function?
